src/pages/oss-friends.js

```diff
@@ -1,150 +1,64 @@
-    href: 'https://openbb.co',
-  },
-  {
-    name: 'Sniffnet',
-    description:
-      'Sniffnet is a network monitoring tool to help you easily keep track of your Internet traffic.',
-    href: 'https://www.sniffnet.net',
-  },
-  {
-    name: 'Typebot',
-    description:
-      'Typebot gives you powerful blocks to create unique chat experiences. Embed them anywhere on your apps and start collecting results like magic.',
-    href: 'https://typebot.io',
-  },
-  {
-    name: 'Webiny',
-    description:
-      'Open-source enterprise-grade serverless CMS. Own your data. Scale effortlessly. Customize everything.',
-    href: 'https://www.webiny.com',
-  },
-  {
-    name: 'Webstudio',
-    description: 'Webstudio is an open source alternative to Webflow',
-    href: 'https://webstudio.is',
-  },
-];
-
 const OSSFriends = () => {
+  const { ossFriends } = usePluginData('oss-friends-plugin');
+
   return (
     <Layout title={title} description={description}>
       <section className="page__section">
         <TitleHeader title={title} />
         <div className="container">
           <div>
-            {friends.length > 0 &&
-              Array.from({ length: Math.ceil(friends.length / 3) }, (_, i) => (
-                <div className="row" key={`row${i}`}>
-                  {friends.slice(i * 3, (i + 1) * 3).map((friend) => (
-                    <div
-                      className="col col--4"
-                      style={{ marginBottom: '20px' }}
-                    >
-                      <div className="col-demo">
-                        <div className="card-demo" style={{}}>
-                          <div className="card">
-                            <div className="card__header">
-                              <h3
-                                className="text--center"
-                                style={{ whiteSpace: 'pre-line' }}
-                              >
-                                {friend.name}
-                              </h3>
-                            </div>
-                            <div className="card__body">
-                              {friend.description}
-                            </div>
-                            <div className="card__footer">
-                              <Link
-                                className={clsx(
-                                  'button button--primary button--block'
-                                )}
-                                href={friend.href}
-                              >
-                                Learn More
-                              </Link>
+            {ossFriends?.length > 0 &&
+              Array.from(
+                { length: Math.ceil(ossFriends.length / 3) },
+                (_, i) => (
+                  <div className="row" key={`row${i}`}>
+                    {ossFriends.slice(i * 3, (i + 1) * 3).map((friend) => (
+                      <div
+                        className="col col--4"
```

Comment:
@niwsa Can you add a min height to the card so that all cards are the same height? I think it looks better that way.